backends/backend_openssl3.c

```diff
@@ -2963,6 +2963,10 @@ static int openssl_rsa_keygen(struct rsa_keygen_data *data,
 					&data->p));
 	CKINT(openssl_pkey_get_bn_bytes(key, OSSL_PKEY_PARAM_RSA_FACTOR2,
 					&data->q));
+	// add test result fields that is required by keyFormat "crt"
```

Comment:
These fields should only be set for the CRT key format. Right now this breaks vector processing for the standard key format.